tests/python/mkl/test_subgraph.py

```diff
@@ -659,9 +659,9 @@ def test_pos_single_conv():
 @with_seed()
 def test_pos_conv_act():
   act_list = {"relu": True,
-              "sigmoid": False,
-              "tanh": False,
-              "softrelu": False,
+              "sigmoid": True,
```

Comment:
Don't change this as `sigmoid` `tanh` and `softrelu` don't support quantization at moment.

Reply:
can it be handled by graph pass?